### DISCUSSION TITLE
Need help on populating a table

### DISCUSSION THREAD

**Discussion by FashMuyhee** [adonisjs/core#1433] [Help] [answered]
Hi guys, i want to know if there is a means of populating a table using database seeder with a custom data not a faker

**Comment by jrishabh55** [comment, marked as answer]
Seeds as described in the docs are just classes with run method. You can put any database related queries in them, You can use seeds to upload your custom data

https://adonisjs.com/docs/4.1/seeds-and-factories#_seeds

I think you are confusing seeds with factories. Factories are an additional resource you can use to create dummy data, You should be able to use seeds alone to upload the data 

```js
const Factory = use('Factory')
const Database = use('Database')

// Get your data from a json file or hardcode it like below

class UserSeeder {
  async run () {
    await Database
        .from('users')
        .insert([{username: 'foo'}, {username: 'bar'}])
  }
}

module.exports = UserSeeder
```

**Reply by FashMuyhee** [reply]
Thanks @jrishabh55  for the reply  will try it out .
